src/SFA.DAS.AANHub.Data/Repositories/CalendarEventsReadRepository.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using SFA.DAS.AANHub.Domain.Models;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.AANHub.Data.Repositories;
+
+[ExcludeFromCodeCoverage]
+internal class CalendarEventsReadRepository : ICalendarEventsReadRepository
+{
+	private readonly AanDataContext _aanDataContext;
+	public CalendarEventsReadRepository(AanDataContext aanDataContext) => _aanDataContext = aanDataContext;
+
+	public async Task<List<CalendarEventModel>> GetCalendarEvents(Guid memberId)
+	{
+		FormattableString sql = $@"select	
+                            CE.Id, 
+	                        C.CalendarName,
+	                        CE.EventFormat, 
+	                        CE.StartDate as [Start], 
+	                        CE.EndDate as [End],
+	                        CE.[Description], 
+	                        CE.Summary,
+	                        CE.[Location],   
+	                        CE.Postcode,
+	                        CE.Latitude,
+	                        CE.Longitude,
+	                        CASE  WHEN (MPLat.ProfileValue is null OR ltrim(MPLat.ProfileValue) = '') THEN null
+			                    WHEN (MPLon.ProfileValue is null OR ltrim(MPLon.ProfileValue) = '') THEN null
+			                    WHEN (CE.Latitude is null OR CE.Longitude is null) THEN null
+		                    ELSE
+			                    geography::Point(CE.Latitude, CE.Longitude, 4326)
+					                .STDistance(geography::Point(convert(float,MPLat.ProfileValue), convert(float,MPLon.ProfileValue), 4326)) * 0.0006213712 END
+					        as Distance,
+	                        CASE WHEN (A.IsActive is null) then convert(bit, 0)
```

Comment:
`ISNULL(A.IsActive, 0)` should do the job

Reply:
ok will do